src/Logging/Logger.php

```diff
@@ -126,8 +126,8 @@ public function delete(array $options = [])
      *
      *     @type string $filter An [advanced logs filter](https://cloud.google.com/logging/docs/view/advanced_filters).
      *     @type string $orderBy How the results should be sorted. Presently,
-     *           the only permitted values are `timestamp asc` (default) and
-     *           `timestamp desc`.
+     *           the only permitted values are `timestamp asc` and `timestamp desc`.
+     *           **Defaults to** `(string) timestamp asc`.
```

Comment:
I had played around with type casts before I settled on the rules I detailed above. I went back through and changed everything (I thought), but if you see any other odd casts I might have missed let me know.